automation/check-patch.unit-test.sh

```diff
@@ -9,6 +9,8 @@ main() {
     cd ${TMP_PROJECT_PATH}
     make bump-all
     make check
+    make generate-doc
```

Comment:
I am good with it, hopefully Quique as well
can you please make those 2 lines a function ? so it will be known they are related